docs/topics/api/overview.rst

```diff
@@ -178,14 +178,12 @@ If only a string is supplied, it will only be used to translate the field in
 the current language.
 
 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Alternate API (v4dev) behavior
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^
+v5 API behavior
+^^^^^^^^^^^^^^^
 
-On the addons-dev.allizom.org (dev) and addons.allizom.org servers (stage) servers
-an additional API version, `v4dev` is available. `v4dev` is not available on
-production AMO (addons.mozilla.org). In `4dev` the response if the ``lang``
-parameter is passed is an object only containing that translation is returned.
+In the experimental :ref:`v5 API <api-experimental-v5>` the response if the ``lang`` parameter is passed
```

Comment:
How about putting this in a changelog format so that it matches that of `v4`? That would make it easier to scan, especially when more breaking changes are added.

Reply:
addressed in https://github.com/mozilla/addons-server/pull/11369